### PR TITLE
Add CLI support for jsx pragmas

### LINKED ISSUE #319: Support jsxPragma and jsxFragmentPragma via command line
Unless I'm totally missing something, it seems like the `jsxPragma` and `jsxFragmentPragma` options are never parsed as options by `sucrase` command. As far as I know, the only way to use these options is by manually writing your own script and passing them into the `transform` function.

### PR DESCRIPTION
**Pull request by jozanza**
closes #319

### CODE REVIEW

File: src/cli.ts [hunk 1]
```diff
@@ -29,6 +29,8 @@ export default function run(): void {
       "Use default TypeScript ESM/CJS interop strategy.",
     )
     .option("--enable-legacy-babel5-module-interop", "Use Babel 5 ESM/CJS interop strategy.")
+    .option("--jsxPragma <string>", "Element creation function, defaults to `React.createElement`")
```
[the remark this answers is not in the document]
Comment (jozanza) [reply src/cli.ts]: No problem. Also, this is my first inline suggestions experience too. It was everything I dreamed of and more 😂 Thanks!

[the remark this answers is not in the document]
Comment (alangpierce) [reply src/cli.ts]: Well, might be nice if I could suggest two lines of changes in the same comment, or if you could apply them in the same commit, but I left feedback about that. 😄

File: src/cli.ts [hunk 2]
```diff
@@ -57,6 +59,8 @@ export default function run(): void {
       transforms: commander.transforms.split(","),
       enableLegacyTypeScriptModuleInterop: commander.enableLegacyTypescriptModuleInterop,
       enableLegacyBabel5ModuleInterop: commander.enableLegacyBabel5ModuleInterop,
+      jsxPragma: commander.jsxPragma || "React.createElement",
+      jsxFragmentPragma: command.jsxFragmentPragma || "React.Fragment",
```
Comment (alangpierce) [inline src/cli.ts]: Oh, build failure actually was caused by this commit. Just a typo, `command` -> `commander.`
```suggestion
      jsxFragmentPragma: commander.jsxFragmentPragma || "React.Fragment",
```

Comment (alangpierce) [reply src/cli.ts]: Just assigned the PR to myself so I can apply the suggestion and see if it fixes things!

Comment (alangpierce) [reply src/cli.ts]: Ah, nevermind, I can't push to your fork. Would be good for you to apply when you get a chance, hopefully will just fix things!

screen shot 2018-10-20 at 2 17 58 pm

**Pull request closed without merging**